xbmc/PlayListPlayer.h

```diff
@@ -103,6 +103,11 @@ class CPlayListPlayer : public IMsgTargetCallback
    */
   int GetCurrentPlaylist() const;
 
+  /*! \brief Get currently active playlist or a default for current player
```

Comment:
> Keeping the logic in a function would make it easier to understand the code

obviously not. if you can't put it easily into words, it is not good. 

> prevent that future changes lead to inconsistent behavior

that's no# reason for dead and ugly code. in 99% of all cases those future changes never happen. code should only reflect the current needs, you never know what the future brings.

please drop the method and move the logic directly to the place where it is needed.